www/gatsby-config.js

```diff
@@ -69,7 +69,9 @@ module.exports = {
         showSpinner: false,
       },
     },
-    `gatsby-plugin-emotion`,
+    {
+      resolve: `gatsby-plugin-emotion`,
+    },
```

Comment:
Not a problem, but I'm curious where this change came from.

Reply:
I experimented with config, in the end removed it but do not revert to the original syntax

Reply:
It occurred that the default settings is exactly what I need. I will revert it to the one-line sytax